waku/v2/protocol/waku_store/README.md

```diff
@@ -0,0 +1,3 @@
+# Waku Store protocol
+
+The store protocol implements historical message support. See https://github.com/vacp2p/specs/blob/master/specs/waku/v2/waku-store.md for more information.
```

Comment:
Here we can add pagination and code structure documentation etc later on (not in this PR)